projects/generate-glitch-dataset/generate_glitches.py

```diff
@@ -63,18 +113,17 @@ def generate_glitch_dataset(
         # get path for this gps day 
         omicron_path = omicron_dir + f'{day}/'
         
-        # do we want pre or post vetoes? I presume we want to use some glitches found in vetoes (CAT3, not CAT1)
         trig_file = glob.glob(omicron_path + f'/*/PostProc/unclustered/triggers_unclustered_{ifo}.txt')[0]
```

Comment:
consider more general path behavior here (i.e. either `os.path.join` or `Path(path_name) / "child_dir" / "file"`). Also consider documenting the expected structure of these directories, and maybe add a quick comment explaining what the structure of this file is so it makes the ensuing code using it more clear